src/components/App/ModalApp.tsx

```diff
@@ -226,6 +234,17 @@ class ModalApp extends Component<Props> {
     }
   }
 
+  setDecoupleFromAPIIfClientHasFeature = (
+    isValidEnterpriseFeature: boolean
+  ) => {
+    if (isValidEnterpriseFeature) {
+      this.props.actions.setDecoupleFromAPI(true)
+    } else {
+      this.props.actions.setDecoupleFromAPI(false)
+      this.onInvalidEnterpriseFeatureException('decouple')
```

Comment:
This feature name string for the error tracking should be updated to either `'network decouple'` or something closer to the renamed configuration IMO.

Reply:
Thanks, I missed this. I've changed it to match the new configuration name.